ord_schema/message_helpers.py

```diff
@@ -371,7 +371,7 @@ def get_reaction_smiles(message, allow_incomplete=True, validate=True):
         if identifier.type == reaction_pb2.ReactionIdentifier.REACTION_SMILES:
             return identifier.value
     reactants, agents, products = set(), set(), set()
-    roles = reaction_pb2.Compound.ReactionRole()
+    roles = reaction_pb2.ReactionRole()
```

Comment:
nit; the enum values are class attributes, not instance attributes; so use `roles = reaction_pb2.ReactionRole` here instead of creating an instance of `ReactionRole`.

Reply:
Fixed. Thanks!